### DISCUSSION TITLE
Unsupported URL scheme

### DISCUSSION THREAD

**Discussion by narekaspaturyan** [webpack/webpack#18878] [Q&A]
Hi, recently I updated my package from 13.2.2 to 16.0.2 and I am geting this error when trying to see styes from source 
```Could not load content for webpack://./components/assets/desktop/_nav.scss (Fetch through target failed: Unsupported URL scheme; Fallback: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME)```, I do not get this error in version 14.2.1

**Comment by alexander-akait** [comment]
Can you provide configuration?

**Reply by narekaspaturyan** [reply]
```
const path = require('path');

const staticStyleFromPath =path.resolve(directories[currentProject], '../common/static')
const isBuild = process.env.BUILD === 'true';
```

```
   {
        test: /\.(scss|css)$/,
        exclude: [staticStyleFromPath, /\.lazy\.(scss|css)$/],
        use: [
            {
                loader: 'style-loader',
                options: { injectType: 'autoStyleTag' },
            },
            {
                loader: 'css-loader',
                options: {
                    sourceMap: !isBuild,
                    importLoaders: 2,
                },
            },
            {
                loader: 'postcss-loader',
                options: { sourceMap: !isBuild },
            },
            {
                loader: 'sass-loader',
                options: { sourceMap: !isBuild },
            },
        ],
    },
    {
        test: /\.lazy\.(scss|css)$/,
        use: [
            {
                loader: 'style-loader',
                options: { injectType: 'lazyAutoStyleTag' },
            },
            {
                loader: 'css-loader',
                options: {
                    sourceMap: !isBuild,
                    importLoaders: 2,
                },
            },
            {
                loader: 'postcss-loader',
                options: { sourceMap: !isBuild },
            },
            {
                loader: 'sass-loader',
                options: { sourceMap: !isBuild },
            },
        ],
    },
```

**Reply by alexander-akait** [reply]
Can you provide the full configuration?

**Reply by narekaspaturyan** [reply]
I can not provide the whole config, is there any specific config you need ?

**Reply by alexander-akait** [reply]
Do you use `source-map-loader` - https://github.com/webpack-contrib/source-map-loader?

**Reply by narekaspaturyan** [reply]
no

**Reply by alexander-akait** [reply]
Can you show me `components/assets/desktop/_nav.scss`?

**Reply by narekaspaturyan** [reply]
```
.lobby {
    &-nav {
        width: 100%;
        padding-top: 23px;

        &-row {
            min-width: 100%;
            display: flex;
            padding-bottom: 7px;

            @include media-md-max {
                padding-right: 16px;
                padding-left: 16px;
            }
        }
    }
}


```

**Reply by alexander-akait** [reply]
Do you see it in a browser console?

**Reply by narekaspaturyan** [reply]
I go to Elements bar click on random element and see the styles and the file that is accosiated with the style at the upper right corner of that block. 
I can see the .css files when clicking on them (style.css?version=2024_10_24_10_7_38)  
but when clicking on files .scss (_nav.scss   or _variables.scss) Could not load content for webpack ... error is shown.

**Reply by alexander-akait** [reply]
It looks like there are original maps but no files. without other information - why and how I can not say, this is not a bug, you can ignore it

**Comment by LyraHarring** [comment]
Understood. Since it's not a bug, I’ll ignore the [ERR_UNKNOWN_URL_SCHEME](https://cheapsslweb.com/resources/how-to-resolve-err_unknown_url_scheme-error) warning for now. It seems the source maps exist, but the actual .scss files are missing, which causes this in devtools. If it becomes distracting, I might disable source maps temporarily in devtools settings.
